src/auth/auth-store.tsx

```diff
@@ -205,12 +204,9 @@ export const useAuthStore = create<AuthState>()((set, get) => {
 				throw error;
 			}
 
-			alert(
-				useI18nStore
-					.getState()
-					.i18n()
-					.navbar.profile.settings.confirmEmail(email),
-			);
+			(
+				document.getElementById("alert-dialog") as HTMLDialogElement
+			).showModal();
```

Comment:
This was a mistake from the past (my bad!) but relooking at this I think we should rather put these 2 lines (that interact with the DOM) outside of the register function and rather below their execution call (e.g. in the submit, after the `register()` is called).

Reply:
Makes sense, I did the same for updatePassword and forgotPassword